data-prepper-plugins/service-map-stateful/src/main/java/org/opensearch/dataprepper/plugins/processor/ServiceMapProcessorConfig.java

```diff
@@ -16,7 +16,7 @@
 public class ServiceMapProcessorConfig {
     private static final String WINDOW_DURATION = "window_duration";
     static final int DEFAULT_WINDOW_DURATION = 180;
-    static final String DEFAULT_DB_PATH = "data/service-map/";
+    static final String DEFAULT_DB_PATH = "/tmp/data/service-map/";
```

Comment:
I don't think this is the right solution. In fact, we've had issues in the past where Data Prepper is trying to write to directories it does not have access to such as `/tmp`.

Maybe, make this configurable instead?

Reply:
Thanks for suggestion! made the change